vcs/src/main/java/org/openjdk/skara/vcs/hg/HgRepository.java

```diff
@@ -50,6 +50,8 @@ public class HgRepository implements Repository {
     private final Path dir;
     private final Logger log = Logger.getLogger("org.openjdk.skara.vcs.hg");
 
+    private static final Hash NULL_REVISION = new Hash("0000000000000000000000000000000000000000");
```

Comment:
Maybe it is better to use `“0”.repeat(40)` to construct such string so that the unnecessary count error can be avoided. But it is also ok now.

Reply:
> Maybe it is better to use `“0”.repeat(40)` to construct such string so that the unnecessary count error can be avoided. But it is also ok now.

Ah, knowing this function for the first time!  Thanks!  I will use this function.